src/vellum/workflows/nodes/displayable/code_execution_node/utils.py

```diff
@@ -14,17 +14,20 @@
 def read_file_from_path(
     node_filepath: str, script_filepath: str, context: Optional[WorkflowContext] = None
 ) -> Union[str, None]:
-    node_filepath_dir = os.path.dirname(node_filepath)
-    full_filepath = os.path.join(node_filepath_dir, script_filepath)
```

Comment:
[nit] still looks like we could keep these two centralized here and strip out namespace within the if block